sats-dna/src/main/kotlin/com/sats/dna/colors/SatsColors.kt

```diff
@@ -273,6 +273,7 @@ class SatsColors(
             val primary: ColorSet,
             val secondary: ColorSet,
             val featured: ColorSet,
+            val positive: ColorSet,
```

Comment:
Since this is under the Indicator Tag section in Figma, I've added this there as part of https://github.com/sats-group/sats-dna-android/pull/437.